test/Evaluate/real.cpp

```diff
@@ -398,7 +400,9 @@ void subsetTests(int pass, Rounding rounding, std::uint32_t opds) {
       ("%d IsInfinite(0x%jx)", pass, static_cast<std::intmax_t>(rj));
 
       static constexpr int kind{REAL::bits / 8};
-      std::stringstream ss, css;
+      std::string ssBuf, cssBuf;
+      llvm::raw_string_ostream ss {ssBuf};
+      llvm::raw_string_ostream css {cssBuf};
```

Comment:
Make sure to run clang-format. (I believe it will squish out all whitespace around the braces.)

Reply:
So, I did not removed iostream from this file and others because it did not interact with the other files that I needed to change. This file has this function:

Reply:
Done